packages/context-menu/package.json

```diff
@@ -32,6 +32,7 @@
     "polymer"
   ],
   "dependencies": {
+    "@open-wc/dedupe-mixin": "^1.3.0",
```

Comment:
Dedupe mixin is only used in the type definitions so it should go to devDependencies instead?

Reply:
The typings are used by consumers so it should be listed under `dependencies` - otherwise it will fail the compilation. I think we can change this by exporting `Constructor` type from `component-base`.